src/jumpstart/JumpstartSendConfirmation.tsx

```diff
@@ -25,17 +31,37 @@ type Props = NativeStackScreenProps<StackParamList, Screens.JumpstartSendConfirm
 const TAG = 'JumpstartSendConfirmation'
 
 function JumpstartSendConfirmation({ route }: Props) {
-  const { tokenId, sendAmount } = route.params
+  const { tokenId, sendAmount, serializablePreparedTransactions } = route.params
   const parsedAmount = new BigNumber(sendAmount)
   const { t } = useTranslation()
+  const dispatch = useDispatch()
 
   const token = useTokenInfo(tokenId)
   const usdToLocalRate = useSelector(usdToLocalCurrencyRateSelector)
   const localCurrencyCode = useSelector(getLocalCurrencyCode)
+  const jumpstartSendStatus = useSelector(jumpstartSendStatusSelector)
+
+  const [showError, setShowError] = useState(false)
```

Comment:
what's the reason of having this local state instead of relying on redux one?

Reply:
it's so that you can dismiss the notification without re-submitting the transaction. the alternative is we can create another action that changes the redux status back to 'idle'...

Reply:
no, we only need it for displaying the error indeed. i've added a new redux action, it's indeed cleaner to rely on one place for the source of truth